modules/va_notify/app/controllers/va_notify/callbacks_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module VANotify
+  class CallbacksController < VANotify::ApplicationController
+    include ActionController::HttpAuthentication::Token::ControllerMethods
+
+    service_tag 'va-notify'
+
+    skip_before_action :verify_authenticity_token, only: [:create]
```

Comment:
VANotify is a trusted system and we will be validating all incoming requests (to this endpoint) with a token that VANotify will provide (`before_action :authenticate_callback`). Because of this it should be okay to skip the `verify_authenticity_token` action.